syft/pkg/cataloger/kernel/package.go

```diff
@@ -18,6 +19,7 @@ func newLinuxKernelPackage(metadata pkg.LinuxKernel, archiveLocation file.Locati
 		PURL:      packageURL(linuxKernelPackageName, metadata.Version),
 		Type:      pkg.LinuxKernelPkg,
 		Metadata:  metadata,
+		CPEs:      []cpe.CPE{cpe.Must("cpe:2.3:o:linux:linux_kernel:*:*:*:*:*:*:*:*", cpe.NVDDictionaryLookupSource)},
```

Comment:
Is it worth trying to put a version in the CPE? I don't know much about how CPEs are used downstream or whether we usually put version info in them, or maybe some other part of syft will. But we're setting a version on the package, so it would make sense to me to set a version on the CPE. But maybe I'm missing something.

Reply:
Yes, it needs to set the version, thanks!